experiment/builder_runner.py

```diff
@@ -304,6 +306,21 @@ def build_and_run(self, generated_project: str, target_path: str,
     if not self._pre_build_check(target_path, build_result):
       return build_result, None
 
+    try:
+      return self.build_and_run_local(generated_project, target_path, iteration,
+                                      build_result)
+    except Exception as err:
+      logging.warning(
+          'Error occurred when building and running fuzz target locally'
+          '(attempt %d): %s', iteration, err)
+      traceback.print_exc()
```

Comment:
Should we use `format_exc()` instead and use `logging` for this?

Reply:
Done!